MCTS/search.cpp

```diff
@@ -73,18 +69,13 @@ action_t SearchTree::search(percept_t prev_obs,
   for (unsigned int sim = 1; sim <= agent->numSimulations(); sim++) {
     reward_t r = rootNode->sample(m);
   }
-  
-  // std::cout << "expected reward[0]: " << rootNode->getChildren()[0]->getSampleMean() <<std::endl;
-  // std::cout << "expected reward [1]: " << rootNode->getChildren()[1]->getSampleMean() <<std::endl;
-  std::cout << "Best Action: " << rootNode->bestAction() << std::endl;
```

Comment:
@Arie3301 Could you just enable these two lines(79,80), at the current stage we need this to have some feedback while testing to see how the learning is progressing.